raiden/messages/cmdid.py

```diff
@@ -18,7 +18,7 @@ class CmdId(enum.Enum):
     REVEALSECRET = 11
     DELIVERED = 12
     LOCKEXPIRED = 13
-    TODEVICE = 14
+    REACHABILITYNOTIFICATION = 14
```

Comment:
Can this be split with `_`, for consistency?
```suggestion
    REACHABILITY_NOTIFICATION = 14
```